Testscripts/Windows/WORKLOAD-HB.ps1

```diff
@@ -5,41 +5,73 @@
 	Perform a simple VM hibernation in Azure
 	This feature might be available in kernel 5.7 or later. By the time,
 	customized kernel will be built.
-	# Hibernation will be supported in the general purpose VM with max 16G vRAM
-	# and the GPU VMs with max 112G vRAM.
+	Hibernation will be supported in the general purpose VM with max 16G vRAM
+	and the GPU VMs with max 112G vRAM.
```

Comment:
do you mean it doesn't work when general purpose vRAM > 16G, GPU VMs > 112G vRAM

Reply:
That is the requirement of Azure hibernation.

Reply:
I can't understand it is the minimal requirement or max requirement by your comments

Reply:
> That is the requirement of Azure hibernation.

I can't understand it is the minimal requirement or max requirement by your comments

Reply:
The line number 8 and 9 tell "max" RAM size. So we do not support a bigger VM size for this feature.